packages/core/tsconfig.json

```diff
@@ -15,6 +15,9 @@
     // "outDir": "dist",
     // go to js file when using IDE functions like
     // "Go to Definition" in VSCode
-    "declarationMap": true
+    "declarationMap": true,
+    "esModuleInterop": true,
+    "moduleResolution": "NodeNext",
+    "module": "NodeNext"
```

Comment:
Do you intend to migrate to JS completely at some point?

Reply:
Not in a near future at least. A lot of things more important needs to be done before.